File: python/models/profissional.py
#definição da classe a ser usada
class Profissional():
    def __init__(self, urgencia: int, nome: str, endereco: str, contato: str, profissao: str):
        self.urgencia = urgencia
        self.nome: str = nome
        self.endereco: str = endereco
        self.profissao: str = profissao
        self.contato: str = contato

    #permite que retorne um valor String quando o objeto for printado, ao invés do endereço de memória
    def __repr__(self):
        return f"Nome: {self.nome}, Endereço: {self.endereco}, Contato: {self.contato}, Profissão: {self.endereco}"

File: java/models/Profissional.java
package models;

public class Profissional {

    private int urgencia;
    private String nome;
    private String endereco;
    private String contato;
    private String profissao;

    public Profissional(int urgencia, String nome, String endereco, String contato, String profissao) {
        this.urgencia = urgencia;
        this.nome = nome;
        this.endereco = endereco;
        this.contato = contato;
        this.profissao = profissao;
    }

    public int getUrgencia() {
        return urgencia;
    }

    public void setUrgencia(int urgencia) {
        this.urgencia = urgencia;
    }

    public String getNome() {
        return nome;
    }

    public String getEndereco() {
        return endereco;
    }

    public String getContato() {
        return contato;
    }

    public String getProfissao() {
        return profissao;
    }

    public void setProfissao(String profissao) {
        this.profissao = profissao;
    }

    @Override
    public String toString() {
        return "Nome: " + nome +
               ", Endereço: " + endereco +
               ", Contato: " + contato +
               ", Profissão: " + profissao +
               ", Urgência: " + urgencia;
    }
}
